### ISSUE TITLE
up running twice for each migration D: wat

### ISSUE THREAD

**Issue by lfreneda** [db-migrate/node-db-migrate#328]
Hello there, anyone faced this buggy?

Each migration is running twice, am I doing something wrong? D:

![db-migrate-running-twice-each](https://cloud.githubusercontent.com/assets/372420/11338072/f515c704-91d8-11e5-8b84-677e72648812.png)

**Comment by wzrdtales** [comment]
I can't help you this way...

Please provide everything that is needed to reproduce your issue:

- used node version
- used db-migrate version
- output from the verbose run
- if needed examples of your migrations, naturally with truncated details if needed

**Comment by lfreneda** [comment]
hello @wzrdtales, thanks for your quick reply D:

Not sure if is really a bug D: db-migrate is smarter than me HAHAHA

Figured out what is going on here while creating a repository to replicate the "problem"(https://github.com/lfreneda/db-migrate-issue-328)

Let me try to explain.

Since I'm using coffeescript (<3) and my IDE is watching coffee files to transpile into javascript, db-migrate finds two versions of each migration (coffee version + javascript version) and then running both.

What I did in order to "fix" was stop watching coffee files under migrations folder~ TA-DA!

Sorry to bother u, thanks a lot
